packages/dashboard/src/pages/CreateMaskWallet/en.html

```diff
@@ -4,7 +4,8 @@
         <meta charset="UTF-8" />
         <meta
             name="viewport"
-            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0" />
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+        />
```

Comment:
You need to upgrade you local prettier to `2.4.0`, and format files with it to revert this kind of change.